contrib/gorilla/websocket/websocket.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+// Package websocket provides tracing functions for tracing the gorilla/websocket
+// package (https://github.com/gorilla/websocket).
+package websocket // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/websocket"
+
+import (
+	"context"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/gorilla/websocket"
+)
+
+// Conn is the wrapper type of a websocket connection.
+type Conn struct {
+	*websocket.Conn
+	ctx context.Context
+}
+
+const (
+	messageTypeTag   = "websocket.message_type"
+	messageLengthTag = "websocket.message_length"
```

Comment:
We usually do it the other way around (`tag*`) shared prefix... Do you mind changing?